lib/validation_profiler/version.rb

```diff
@@ -1,5 +1,5 @@
 # Namespace
 module ValidationProfiler
   # :nodoc:
-  VERSION = '1.6.0'.freeze
+  VERSION = '1.6.1'.freeze
```

Comment:
This is more than a bug fix, should it be 1.7.0 instead?

Reply:
Yeah you're right. I'll update this.